src/gpg2john.c

```diff
@@ -346,18 +349,24 @@ int gpg2john(int argc, char **argv)
 		--argc;
 	}
 	if (argc < 2) {
-		fprintf(stderr, "Usage: %s [-d] [-S] <GPG Secret Key File(s)>\n", argv[0]);
+		fprintf(stderr, "Usage: %s [-d] [-S] <GPG Secret Key File>\n", argv[0]);
 		fprintf(stderr, "   if -d is used, then debugging of the object types decoded is written\n");
 		fprintf(stderr, "   if -S is used, then subkeys will also be output\n");
 		exit(-1);
 	}
 
-	for (i = 1; i < argc; ++i) {
+	// for (i = 1; i < argc; ++i) {
+	for (i = 1; i < 2; ++i) {
```

Comment:
This is just silly. Fix the bug instead.

Reply:
If there even is one 😆

Reply:
```
$ ../run/gpg2john key.txt key.txt
Adobe PSIRT:$gpg$*1*1308*4096*d...75e98cdbbbc96:::Adobe PSIRT <psirt@adobe.com>::key.txt
unexpected end of file.
```

The `unexpected end of file` message is what I meant by "fails".

[key.txt](https://github.com/magnumripper/JohnTheRipper/files/1343855/key.txt) <-- test using this file.